apps/mail/components/create/schedule-send-picker.tsx

```
@@ -0,0 +1,107 @@
+                return format(parsed, 'dd MMM yyyy hh:mm aaa');
+              } catch (error) {
+                console.error('Error formatting date', error);
+                return 'Send later';
+              }
+            })()}
+          </span>
+        </button>
+      </PopoverTrigger>
+      <PopoverContent className="z-[100] w-64 p-4" align="start" side="top" sideOffset={8}>
+        <div className="flex flex-col gap-4">
+          <label className="text-sm font-semibold">Choose date & time</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)